src/generator-enums.cs

```diff
@@ -161,6 +161,18 @@ void GenerateEnum (Type type)
 				print ("");
 			}
 			
+			print ("public static NSString GetConstant (this {0}? self)", type.Name);
```

Comment:
that's a lot more metadata (and dupe code) for a null check
and if we make it public then we can't remove it in the future
can we make-do without it ?